packages/ruleset/package.json

```diff
@@ -22,18 +22,18 @@
   },
   "dependencies": {
     "@ibm-cloud/openapi-ruleset-utilities": "1.3.1",
-    "@stoplight/spectral-formats": "^1.5.0",
+    "@stoplight/spectral-formats": "^1.6.0",
```

Comment:
Seemed like a good time to bump some of our dependencies.  This addressed the "ip"-related vulnerability that `npm install` was reporting.